lib/galaxy/webapps/galaxy/api/dependencies.py

```diff
@@ -0,0 +1,91 @@
+"""
+This module *does not* contain API routes. It exclusively contains dependencies to be used in FastAPI routes
```

Comment:
Maybe this stuff can go in `lib/galaxy/webapps/galaxy/api/__init__.py` ? That would be less confusing, I think.

Reply:
Yep, done that for now. I suppose this will grow significantly, but I think that's a good location for now.